config/src/config/network_config.rs

```diff
@@ -41,13 +41,9 @@ pub struct NetworkConfig {
     pub mutual_authentication: bool,
     // Leveraged by mutual_authentication for incoming peers that may not have a well-defined
     // network address.
-    #[serde(skip)]
     pub network_peers: NetworkPeersConfig,
-    pub network_peers_file: PathBuf,
     // Initial set of peers to connect to
-    #[serde(skip)]
     pub seed_peers: SeedPeersConfig,
-    pub seed_peers_file: PathBuf,
```

Comment:
Was there a reason the peers / seed_peers were being saved to a different file?

Reply:
When I first started working on configs last year, almost anything that could be in a different file was. It wasn't motivated, it was just done. My take is that we should build the simplest config and improve it later if we need to. But you can see in this PR, removing this aspect removes a lot of code and complexity.